src/index.js

```diff
@@ -160,12 +134,15 @@ function initGa(instanceName, tags) {
  * @param {String} phase the phase to load
  */
 function loadGtm(phase) {
-  // eslint-disable-next-line no-console
-  console.assert(gtm.config.analytics, 'Analytics is disabled in the martech config');
-  if (gtm.config.containers[phase]?.length > 0) {
-    pushToDataLayer({ event: 'gtm.js', [`gtm.${phase}.start`]: new Date().getTime() });
-    gtm.config.containers[phase].forEach((container) => {
-      loadScript(`${GTM_HOST}/gtm.js?id=${container}&l=${gtm.config.dataLayerInstanceName}`);
+  if (!this.config.analytics) {
+    // eslint-disable-next-line no-console
+    console.warn('Analytics is disabled in the martech config');
+    return;
+  }
+  if (this.config.containers[phase]?.length > 0) {
+    this.pushToDataLayer({ event: 'gtm.js', [`gtm.${phase}.start`]: new Date().getTime() });
```

Comment:
Do we maybe also need `gtm.start` triggered? I think this is the default that is expected.
I'd probably trigger it along with `gtm.eager.start`. What do you think?

Reply:
```suggestion
    this.pushToDataLayer({ event: 'gtm.js', [`gtm.${phase}.start`]: Date.now() });
```